docs/0.5/howto/configure_logging.md

```diff
@@ -0,0 +1,230 @@
+`level` field controls the filtering functionality of the logger as discussed
+above.
+
+For non root loggers the `appenders` and `level` fields are optional. The level
+will be set to the root loggers level. Appenders are a little more complicated,
+each logger inherits its parents appenders. So if you had two loggers, `root`
+and `splinter` and `root` has both `stdout` and `stderr` appenders configured.
+Then the `splinter` logger would also have the same `stdout` and `stderr`
+appenders plus any specifically called out in its configuration.
+
+## Appenders in Depth
+
+There are several types of appenders denoted by the `kind` field.
+
+#### Appender Kinds
+* stdout
+* stderr
+* file
+* rolling_file
+
+The `stdout` and `stderr` types log to stdout and stderr respectively as you
+would expect. The `file` and `rolling_file` types both log to files and have
+another required field `filename` that is the path to the desired log file. The
+`rolling_file` type also has a required field `size` that controls when the file
+is rolled up. When the log file reaches the size specified the file is deleted
+and restarted. The files do not need to exist at Splinter startup but the
+containing directory does.
+
+## Example Configurations
+
+### The default config
+
+This is the configuration provided with all defaults written out. Something to
+note is that while the stdout appender does not have a level specified by
+default it can be filled in at runtime with one of `Info`, `Debug`, `Trace` for
+one, two or three plus occurrences of the `-v` verbosity flag.
+
+``` console
+
+[appenders.stdout]
+kind = "stdout"
+encoder = "[{d(%Y-%m-%d %H:%M:%S%.3f)}] T[{T}] {l} [{M}] {m}\n"
+
+[loggers.root]
+appenders = ["stdout"]
+level = "Trace"
+
+[loggers.tokio]
+appenders = ["stdout"]
+level = "Warn"
+
+[loggers.tokio_reactor]
+appenders = ["stdout"]
+level = "Warn"
+
+[loggers.hyper]
+appenders = ["stdout"]
+level = "Warn"
+
+```
+
+### Redirect Splinter Daemon logs to file
+
+This example will redirect the logs from libsplinter and the splinter daemon to
+a single log file. This is the most generally useful configuration besides the
+default configuration because it will seperate the logs related to splinters
+functionality from the logs generated by underlying libraries.
+
+``` console
+
+[appenders.splinter_file]
+kind = "file"
+filename = "/var/log/splinter/splinter.log"
+
+[loggers.splinter]
+appenders = ["splinter_file"]
+level = "Info"
+
+[loggers.splinterd]
+appenders = ["splinter_file"]
+level = "Info"
+
+```
+
+### All Logs All the Time
+
+This example has turned off all log filtering, every log statement will be
+logged to stdout. This is not recommended and may cause performance issues.
+
+``` console
+
+[loggers.root]
+appenders = ["stdout"]
+level = "Trace"
+
+[loggers.tokio]
+level = "Trace"
+
+[loggers.tokio_reactor]
+level = "Trace"
+
+[loggers.hyper]
+level = "Trace"
+
+```
+
+
+## Procedure
+
```

Comment:
Remove the instances of multiple empty lines in a row

Reply:
fixed